src/Events/EventWatcher.php

```diff
@@ -14,6 +14,10 @@ public function register(): void
         Event::listen('*', function (string $eventName, array $data) {
             $event = $data[0] ?? (object) null;
 
+            if(! is_object($event)) {
+                return;
+            }
+
             if (! method_exists($event, 'broadcastOn')) {
```

Comment:
The following would have sufficed as conditionals are evaluated in order and reduced the need for an additional return clause however, the above works too. Thanks!

```suggestion
            if (!is_object($event) || ! method_exists($event, 'broadcastOn')) {
```